dmaws/commands/migratedata.py

```diff
@@ -68,10 +69,7 @@ def dump_to_target(ctx, src_pg_client):
 
     instance = rds.get_instance(plan.get_value('stacks.database.outputs')['URL'])
 
-    instance = rds.allow_access_to_instance(
-        instance, IMPORT_SECURITY_GROUP_NAME,
-        ctx.variables['dev_user_ips'],
-        ctx.variables['vpc_id'])
+    StackPlan.from_ctx(ctx, apps=['database_dev_access']).create()
```

Comment:
I think these are the places you need to pass `profile_name` instead of in `create_scrubbed_instance`.
The original `ctx` will match the profile, since Jenkins takes care of that by setting `AWS_PROFILE` for the run. It's when we switch to the target context that we need to override it with the target stage.